app/Console/AbstractRemoveLabel.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Silex\Console;
+
+use Broadway\CommandHandling\CommandBus;
+use Doctrine\DBAL\Connection;
+use Symfony\Component\Console\Command\Command;
+
+abstract class AbstractRemoveLabel extends Command
+{
+    private Connection $connection;
+
+    protected CommandBus $commandBus;
+
+    public function __construct(Connection $connection, CommandBus $commandBus)
+    {
+        $this->connection = $connection;
+        $this->commandBus = $commandBus;
+        parent::__construct();
+    }
+
+    /*
+     * @return string|false
+     */
+    protected function getLabel(string $labelId)
```

Comment:
This gets the label name, but there is also a repo to get the full label, see https://github.com/cultuurnet/udb3-silex/blob/ea62766e0bd8414d9f364c04f512161430705996/src/Label/ReadModels/JSON/Repository/ReadRepositoryInterface.php#L11